src/cdp/cdpConnection.ts

```diff
@@ -106,7 +106,7 @@ export class CdpConnection {
 
       const messageStr = JSON.stringify(messageObj);
       const messagePretty = JSON.stringify(messageObj, null, 2);
-      this.#transport.sendMessage(messageStr);
+      void this.#transport.sendMessage(messageStr);
```

Comment:
catch too?

Reply:
how do we report errors on a connection? if that fails it would bring down the entire program. I think we need to emit an error on the command, e.g., by invoking the callback.